perceval/backends/askbot.py

```diff
@@ -0,0 +1,534 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2016 Bitergia
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program; if not, write to the Free Software
+# Foundation, Inc., 59 Temple Place - Suite 330, Boston, MA 02111-1307, USA.
+#
+# Authors:
+#     Alberto Martín <alberto.martin@bitergia.com>
+#
+
+import requests
+import json
+import re
+import bs4
+import logging
+
+from ..backend import Backend, metadata, BackendCommand
+from ..utils import urljoin, DEFAULT_DATETIME, str_to_datetime
+
+logger = logging.getLogger(__name__)
+
+
+class AskbotParser:
+
+    def parse_question(self, question, html_question):
```

Comment:
In my opinion, this method should be changed to just parse a HTML question page. Then, in the backend build the item with all the parts (answers, comments, etc). I think the same behaviour must be followed by all the parsing methods.